### PR TITLE
doco: help users identify 2.1-1.6 boards

### PR DESCRIPTION
**Pull request by Tyler-2**
Add section for 2.1-1.6 boards that have alternate markings. As far as I can tell this information doesn't exist anywhere else.

### CODE REVIEW

**Comment by Tyler-2** [comment]
Note on my chip-sticker, it's actually spelled "modle" as mentioned in the doco update.

File: docs/hardware/devices/lora/index.mdx [hunk 1]
```diff
@@ -145,6 +145,12 @@ Early versions of some of these boards contained the wrong component in the LiPo
   - Micro USB
   - Antenna: SMA antenna connector
 
+## Variations of identifying marks
+  - FCC ID: 2ASYE-T3-V1-6-1
+    - "LILYGO 868/915 Modle: LORA32" sticker on the chip
```
Comment (caveman99) [inline docs/hardware/devices/lora/index.mdx:150]: The LORA32 sticker is on different boards too ... it's not exclusive here. Better remove that line.

**Review by caveman99 — CHANGES_REQUESTED** [review]

File: docs/hardware/devices/lora/index.mdx (hunk 1 above)
Comment (Tyler-2) [reply docs/hardware/devices/lora/index.mdx:150]: Hmm, I was trying to capture all of the markings. Like, if it has each of those things, it's a 2.1-1.6. I guess that's not clear though... any suggestions?

Comment (Tyler-2) [reply docs/hardware/devices/lora/index.mdx:150]: Or are you saying that that sticker is so common that there's no point including mention of it at all. No one is likely to find a board with the other markings that _doesn't_ have that sticker on the chip?

Comment (caveman99) [reply docs/hardware/devices/lora/index.mdx]: yes... all lilygo boards with LORA functonality have that particular sticker. or a different one saying 'SX1262' - but that's it. Doesn't really help with ID.

**Comment by caveman99** [comment]
Also the FCC/CE sticker is missing on older boards.

**Review by caveman99 — APPROVED** [review]

**Comment by caveman99** [comment]
Looking good now, however I can't merge that until you sign the CLA...

**Comment by caveman99** [comment]
@Tyler-2 I can't merge the change until you sign the CLA.

**Pull request merged**